server/validate/email/validate.go

```diff
@@ -167,8 +171,8 @@ func (v *validator) Init(jsonconf string) error {
 	if v.Login != "" {
 		v.auth = smtp.PlainAuth("", v.Login, v.SenderPassword, v.SMTPAddr)
 	} else {
-		// SendFrom could be an RFC 5322 address of the form "John Doe <jdoe@example.com>". Parse it.
-		v.auth = smtp.PlainAuth("", v.senderEmail, v.SenderPassword, v.SMTPAddr)
+		// if unset v.Login, skip authentication
+		v.auth = nil
```

Comment:
`v.auth` is `nil` already. This assignment is unnecessary. Please remove.

Reply:
deleted.